test/reporters/dot.spec.js

```diff
@@ -50,7 +56,7 @@ describe('Dot reporter', function() {
         runner = createMockRunner('pending', 'pending');
         Dot.call({epilogue: function() {}}, runner);
         process.stdout.write = stdoutWrite;
```

Comment:
Aren't we doing stream reassignment in `afterEach()` already? Why repeat here?

Reply:
I will test what you provide.